.github/workflows/main.yml

```diff
@@ -90,14 +111,377 @@ jobs:
+        with:
+          name: dist-test-mv2
+          path: ./dist-test-mv2/
+
+  prep-build-test-flask:
+    needs: prep-deps
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout and setup environment
+        uses: metamask/github-tools/.github/actions/checkout-and-setup@1299bb1de0c6974ae6d0a32c7e8897fe168239ac
+        with:
+          is-high-risk-environment: false
+
+      - run: yarn build:test:flask
+
+      - name: Move test build to dist-test-flask
+        run: mv ./dist ./dist-test-flask
+
+      - name: Move test zips to builds-test-flask
+        run: mv ./builds ./builds-test-flask
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: builds-test-flask
+          path: ./builds-test-flask/
+
+      - name: Upload dist
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist-test-flask
+          path: ./dist-test-flask/
+
+  prep-build-test-flask-mv2:
+    needs: prep-deps
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout and setup environment
+        uses: metamask/github-tools/.github/actions/checkout-and-setup@1299bb1de0c6974ae6d0a32c7e8897fe168239ac
+        with:
+          is-high-risk-environment: false
+
+      - run: yarn build:test:flask:mv2
+
+      - name: Move test build to dist-test-flask-mv2
+        run: mv ./dist ./dist-test-flask-mv2
+
+      - name: Move test zips to builds-test-flask-mv2
+        run: mv ./builds ./builds-test-flask-mv2
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: builds-test-flask-mv2
+          path: ./builds-test-flask-mv2/
+
+      - name: Upload dist
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist-test-flask-mv2
+          path: ./dist-test-flask-mv2/
+
+  prep-build-ts-migration-dashboard:
+    needs: prep-deps
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout and setup environment
+        uses: metamask/github-tools/.github/actions/checkout-and-setup@1299bb1de0c6974ae6d0a32c7e8897fe168239ac
+        with:
+          is-high-risk-environment: false
+
+      - run: yarn ts-migration:dashboard:build
+
+      - name: Upload ts-migration-dashboard
+        uses: actions/upload-artifact@v4
+        with:
+          name: ts-migration-dashboard
+          path: ./development/ts-migration-dashboard/build/final/
+
+  test-e2e-chrome-webpack:
+    name: Test e2e chrome webpack
+    #if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/Version-v')
+    #if: github.ref == 'refs/heads/ci-prep-build'
+    needs:
+      - prep-build-test-webpack
+      - get-changed-files-with-git-diff
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        index: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+    env:
+      MATRIX_INDEX: ${{ matrix.index }}
+      MATRIX_TOTAL: 10
+      GITHUB_ACTIONS: true
```

Comment:
It's `GITHUB_ACTION`, not ` GITHUB_ACTIONS`, and it's defined by default (this is also in a few other places)